tools/packaging/kernel/kata_config_version

```diff
@@ -1 +1 @@
-132
+133
```

Comment:
newline at end of file, please fix it.

Reply:
@Apokleos this file doesn't have newline at end of file, PTAL https://github.com/kata-containers/kata-containers/blob/c37feeb8864ab1e90efad0023a288b8ab5e49aa3/tools/packaging/kernel/kata_config_version

Reply:
@Apokleos do you want this PR add a newline at end of file? I saw this file didn't have that before.